website/content/en/docs/deployment/cognito/guide-automated.md

```diff
@@ -82,7 +82,7 @@ This guide assumes you have Python 3.8 installed and that you have completed the
             NAME            CLASS    HOSTS   ADDRESS                                                              PORTS   AGE
             istio-ingress   <none>   *       k8s-istiosys-istioing-xxxxxx-110050202.us-west-2.elb.amazonaws.com   80      15d
             ```
-        2. If `ADDRESS` is empty after a few minutes, check the logs of alb-ingress-controller by following [this guide](/kubeflow-manifests/docs/troubleshooting-aws/#alb-fails-to-provision)
+        2. If `ADDRESS` is empty after a few minutes, check the logs of alb-ingress-controller by following [this guide]({{< ref "/docs/troubleshooting-aws.md#alb-fails-to-provision" >}})
```

Comment:
Need this same change [for LB guide](https://akartsky.github.io/kubeflow-manifests/main/docs/deployment/add-ons/load-balancer/guide/#update-the-domain-with-alb-address)

Text:
If ADDRESS is empty after a few minutes, check the logs of the controller by following the troubleshooting steps in [ALB fails to provision](https://awslabs.github.io/kubeflow-manifests/docs/troubleshooting-aws/#alb-fails-to-provision).